app/api/attendees.py

```diff
@@ -76,36 +89,15 @@ def query(self, view_kwargs):
             query_ = query_.filter_by(event_id=event.id)
         return query_
 
-    def before_create_object(self, data, view_kwargs):
-        """
-        method to create object before post
-        :param data:
-        :param view_kwargs:
-        :return:
-        """
-        if view_kwargs.get('order_id') and view_kwargs.get('ticket_id'):
-            order = safe_query(self, Order, 'id', view_kwargs['order_id'], 'order_id')
-            data['order_id'] = order.id
-
-            ticket = safe_query(self, Ticket, 'id', view_kwargs['ticket_id'], 'ticket_id')
-            data['ticket_id'] = ticket.id
-
-        event = None
-        if view_kwargs.get('event_id'):
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
-        elif view_kwargs.get('event_identifier'):
-            event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'event_identifier')
-        if event:
-            data['event_id'] = event.id
-
     view_kwargs = True
     decorators = (jwt_required,)
```

Comment:
TicketHolder access is required to view the list/detail. Please check.

Also check the permissions in other methods too according to [dev-handbook](https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md#attendees---eg-v1attendees).